src/components/breadcrumbs/template.test.js

```diff
@@ -0,0 +1,112 @@
+/* globals describe, it, expect */
+
+const { render, getExamples } = require('../../../lib/jest-helpers')
+
+const examples = getExamples('breadcrumbs')
+
+describe('Breadcrumbs', () => {
+  describe('by default', () => {
+    it('renders with classes', () => {
+      const { $ } = render('breadcrumbs', {
+        classes: 'app-c-breadcrumbs--custom-modifier'
+      })
+
+      const $component = $('.govuk-c-breadcrumbs')
+      expect($component.hasClass('app-c-breadcrumbs--custom-modifier')).toBeTruthy()
+    })
+
+    it('renders with attributes', () => {
+      const { $ } = render('breadcrumbs', {
+        attributes: {
+          'id': 'my-navigation',
+          'role': 'navigation'
+        }
+      })
+
+      const $component = $('.govuk-c-breadcrumbs')
+      expect($component.attr('id')).toEqual('my-navigation')
+      expect($component.attr('role')).toEqual('navigation')
+    })
+
+    it('renders with items', () => {
+      const { $ } = render('breadcrumbs', {
+        items: [
+          {
+            'text': 'Section 1'
+          },
+          {
+            'text': 'Sub-section'
+          }
+        ]
+      })
+
+      const $items = $('.govuk-c-breadcrumbs__list-item')
+      expect($items.length).toEqual(2)
+    })
+
+    it('renders item with text', () => {
```

Comment:
Can we include a test that tries to put `html` in `text` that asserts it is escaped?

Reply:
Added `renders item with escaped entities in text`